mutters-slots/src/main/java/com/rabidgremlin/mutters/slots/DefaultValueSlot.java

```diff
@@ -4,10 +4,10 @@
  * An interface for a slot that has a default value, for cases when no slot value is found.
  *
  * @author foskic
- *
+ * @author tewsl
  */
 public interface DefaultValueSlot {
 
-    Object getDefaultValue();
+    default Object getDefaultValue() { return ""; }
```

Comment:
I'm not sure it is a good assumption that the default object should be an empty string.